lib/slack-ruby-client.rb

```diff
@@ -15,12 +15,12 @@
 require 'logger'
 begin
   require 'picky'
-rescue LoadError
+rescue LoadError # rubocop:disable Lint/HandleExceptions
   # ignore, only used in users_search
 end
 begin
   require 'openssl'
-rescue LoadError
+rescue LoadError # rubocop:disable Lint/HandleExceptions
```

Comment:
@jcraigk You could either enable the `AllowComments` option on the `Lint/HandleExceptions` cop, or just add a `nil` in front of the comments in the blank exception handler, instead of disabling the cop here.

Reply:
Ah good call.  It seems like the `nil` addition should produce the same results in Rubocop since it's functionally equivalent (the method returns `nil` implicitly on `rescue`)...I wonder if that should be changed on the Rubocop side... 🤔